### DISCUSSION TITLE
Any possibility working on OKD or OpenShift and Ansible?

### DISCUSSION THREAD

**Discussion by benyaminl** [MansoorMajeed/devops-from-scratch#7] [Q&A] [answered]
Hello, just asking, is there any possibility for you to work on OKD or OpenShift? Also Ansible?

Thank you!

**Comment by MansoorMajeed** [comment, marked as answer]
Hey @benyaminl do you mean to create videos on these? I might at some point. Right now, I am trying to get back to resuming the video series, which has been difficult

**Reply by benyaminl** [reply]
Yeah. I just asking. Anyway thanks for responding. Seem you are pretty busy. Its really a good video for all of devops, and even you are providing transcript! Thanks you!
